pkg/build/python-shim.py

```diff
@@ -34,6 +35,21 @@ def run(args):
         {{if .EntrypointFunc}}
         ret = mod.{{.EntrypointFunc}}.base_func(json.loads(args[1]))
         {{else}}
+        if not hasattr(mod, "main"):
+            raise Exception("""Task is missing a `main` function. Add a main method like so and re-deploy:
```

Comment:
Does Python call them methods or functions? Let's be consistent.

```suggestion
            raise Exception("""Task is missing a `main` method. Add the following to your task and re-deploy:
```